content/docs/user-guide/dvc-files.md

```diff
@@ -282,9 +282,7 @@ the file:
 stages:
   build-us:
     cmd: >-
-      python train.py
-      --tresh ${models.us.threshold}
-      --out ${models.us.filename}
+      python train.py --tresh ${models.us.threshold} --out ${models.us.filename}
```

Comment:
> Continues https://github.com/iterative/dvc.org/pull/2054#pullrequestreview-558337058

So we're back on square one @rogermparent 😆. Any other ideas?

Reply:
p.s. see https://github.com/iterative/dvc.org/pull/2060#pullrequestreview-558974903 if you haven't.

Reply:
Not a huge deal BTW, we could go with the "pretty" version. I just thought it would be interesting to show how advanced YAML is supported...

Reply:
does prettier support some meta commands to disable it locally?

Reply:
Yaml accepts a `# prettier-ignore` comment, but we'll have to test out if that translates to frontmatter

Reply:
Roger fixed this particular case in 088fbe8.

Reply:
p.s. related note in https://github.com/iterative/dvc.org/pull/2062#pullrequestreview-559342263